package.json

```diff
@@ -100,15 +100,14 @@
     "react-autocomplete": "1.7.2",
     "react-copy-to-clipboard": "4.3.1",
     "react-dom": "15.6.2",
-    "react-dropzone": "3.13.4",
-    "react-foundation-components": "git+https://github.com/valzav/react-foundation-components.git#d14362c7c8eee946a4acc3b18d70271d5a82813e",
-    "react-highcharts": "8.4.2",
-    "react-intl": "2.4.0",
-    "react-medium-editor": "1.8.1",
-    "react-notification": "5.0.7",
-    "react-overlays": "0.7.2",
-    "react-portal": "2.2.1",
-    "react-prop-types": "0.3.2",
+    "react-dropzone": "^3.7.3",
```

Comment:
please keep the corrected & fixed dependency versions

Reply:
oops yes.